.nvmrc

```diff
@@ -1 +1 @@
-8.15.1
+10.16.3
```

Comment:
We should also bump the version in `package.json` for volta so that development environments are consistent.

Reply:
Ugh, I was already getting tired of the spread I've been tidying up. Incoming fix...